srcpkgs/opencv/template

```diff
@@ -41,9 +41,10 @@ case "$XBPS_TARGET_MACHINE" in
 	ppc*) broken="grfmt_jpeg2000.cpp:380:48: error: lvalue required as unary '&' operand";;
 esac
 
-post_install() {
-	vlicense LICENSE
-}
+#Apache license doesn't require install. Leaving here in case project reverts back to BSD license.
```

Comment:
Is this really necessary ? Going back on this will be easy, better not to leave useless cruft in there, no ?